style.css

```diff
@@ -0,0 +1,53 @@
+body{
+    text-align: center;
+    background-color: grey;
+}
+
+body > h1 >p{
+    color: white;
+    
+}
+
+.form-page{
+    
+    width: 600px;
+    padding: 20px;
+    margin: 50px 30%;
+    background-color: white;
+    text-align: left;
+}
+
+
+
```

Comment:
please, i would suggest that you follow the linters.yml instructions to make required changes, if you are confused, look at the linters.yml report the numbering by your left side begins this way 1. 3: 5
where 1 indicates the serial number, 3 indicate the line number of your code, and 5 indicates the points where the change is required on code line 3 number 1 error
![image](https://user-images.githubusercontent.com/31141344/229635895-5baf196c-cb98-4db9-9c3e-11e8f0e4219a.png)